client/flutter/lib/components/youtube_embed.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+const double youtubeAspectRatio = (16.0 / 9.0);
```

Comment:
The aspect ratio for the "Coronavirus - seven steps to prevent the spread of the virus" video is actually 1:1 so it might be worth making this the default named argument and changing it in the ProtectYourself page